.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--- Add a pull request title above in this format -->
+<!--- real example: 'DAP-2: Design login component' -->
+<!--- 'DAP-JiraIssueNumber: JiraIssueTitle' -->
+
+# Description of What I changed
+
+<!--- Please include a summary of the changes and the related issue.-->
+<!--- It can simply be your commit message, which you must have -->
+
+Issue I worked on
+<!--- This project only accepts pull requests related to open issues -->
+<!--- Want a new feature or change? Discuss it in an issue first -->
+<!--- Found a bug? Point us to the issue/or create one so we can reproduce it -->
+<!--- Just add the issue number at the end: -->
+see https://ampath.atlassian.net/browse/DAP-2
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
This file already exists in the `main` branch. It shouldn't be part of your changes